README.md

```diff
@@ -21,46 +36,158 @@ Or, the same command using `jing`.
 
 # Building MEI
 
-The MEI Source is not a schema in itself; rather, it can be used to build customized schemas. To create your own customization, you should understand the building and customization process.
+The MEI Source is not a schema in itself; rather, it can be used to build customized schemas, such as mei-CMN, mei-Mensural, mei-all, etc. (also see [Customizing MEI](#customizing-mei)).
 
-## Customizing MEI
+Nevertheless it is possible to build any customization locally in your working copy of this repository. In order to do so follow the steps below:
 
-The MEI Schema may be customized to express and validate different types of music documents. These customizations are controlled by customization  By default, MEI includes five customizations:
+1. Create a recursive clone of this repository
 
- * mei-all: The full MEI Schema. This is the most permissive version of MEI.
- * mei-CMN: Validates MEI files that express common Western music notation.
- * mei-Mensural: Validates MEI files that express white Mensural notation (will raise validation errors if elements like "measure" exist in the MEI encoding).
- * mei-Neumes: Validates MEI files that express Neume notation (like Mensural, will raise validation errors if elements that are not part of neume notation exist in an encoding.)
+   * If you do not have a clone on your local machine yet run the following from your command line:
 
-## Why Customizations?
+     ```bash
+     git clone https://github.com/music-encoding/music-encoding.git --recursive
+     ```
+     
+   * If you already have a clone on your system you still might have to initialize the submodules by running the following commands from the command line:
 
-For those who are used to having a single DTD or W3C Schema to validate music notation encodings, the customization process may seem to be a complex way of arriving at a schema to validate music notation. However, customizations are a vital part of the expressive power of MEI, and when used to their full extent, can assist organizations in ensuring the integrity and validity of their data.
+     Switch to your clone's directory:
+     ```bash
+     cd [YOUR-CLONE-LOCATION]
+     ```
+     Initialize the submodules:
+     ```bash
+     git submodule init
+     ```
+     Update the submodules:
+     ```bash
+     git submodule update
+     ```
+2. Check if your system meets the build requisites
 
-When designing a music encoding system there are many contradictory and non-standardized practices associated with writing music notation. Different repertoires may have extremely different ways of expressing pitch or rhythm; for example, rhythm in Mensural notation is incompatible with the later systems developed in common Western notation.
+   * Is Java 8 or above available on your machine?
```

Comment:
What do you do if these are not available?

Reply:
Same case here, @bwbohl. Would you mind taking a look at this one too? 
thank you :)

Reply:
done